### PR TITLE
#Issue KS-0 feat:Added Knowlg Search API and Job

### PR DESCRIPTION
**Pull request by aimansharief**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions: Java 11, scala-2.11, play-2.7.2
* Hardware versions: 2 CPU/ 4GB RAM

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: search-api/knowlg-search/search-service/app/utils/ApiId.scala [hunk 1]
```diff
@@ -7,4 +7,5 @@ object ApiId {
 	final val APPLICATION_SEARCH = "api.search-service.search"
 	final val APPLICATION_PRIVATE_SEARCH = "api.search-service.private.search"
 	final val APPLICATION_COUNT = "api.search-service.count"
+	final val APPLICATION_AUDIT_HISTORY = "api.search-service.audit_history.read"
```
Comment (maheshkumargangula) [inline search-api/knowlg-search/search-service/app/utils/ApiId.scala]:
```suggestion
	final val APPLICATION_AUDIT_HISTORY = "api.knowlg-search.audit-history.read"
```

File: search-job/jobs-core/src/main/scala/org/sunbird/job/cache/local/FrameworkMasterCategoryMap.scala [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+package org.sunbird.job.cache.local
 
 import com.twitter.storehaus.cache.Cache
 import com.twitter.util.Duration
-import org.sunbird.common.Platform
 
 object FrameworkMasterCategoryMap {
 
-  val ttlMS = Platform.getLong("master.category.cache.ttl", 10000l)
+  val ttlMS = 100000l//Platform.getLong("master.category.cache.ttl", 10000l)
```
Comment (maheshkumargangula) [inline search-job/jobs-core/src/main/scala/org/sunbird/job/cache/local/FrameworkMasterCategoryMap.scala:8]: Why are we not using the configuration?

**Review by maheshkumargangula — CHANGES_REQUESTED** [review]
Remove the vscode or the intellij related files.

File: search-api/knowlg-search/search-service/app/utils/ApiId.scala (hunk 1 above)
Comment (aimansharief) [reply search-api/knowlg-search/search-service/app/utils/ApiId.scala:10]: Updated AUDIT_HISTORY ApiId

**Pull request merged**